demos/tests/run_tests.py

```diff
@@ -167,7 +167,7 @@ def main():
 
             with temp_dir_as_path() as temp_dir:
                 arg_context = ArgContext(
-                    source_dir=demos_dir / demo.subdirectory,
+                    source_dir=omz_dir / 'data/datasets_labels',
```

Comment:
That's not right. `source_dir` should point to the demo's source directory (hence the name).

Reply:
done